index.html

```diff
@@ -96,30 +96,39 @@
 
       function checkForStandardMfgs(firstThree) {
         if(-1 == knownMacs.indexOf(firstThree.join(""))){
-          displayWarning(
+          displayError(
             "The MAC address does not appear to be from a PittMesh standard manufacturer (" +
             standardMfgs.join(", ") +
             "). The IP address calculation may be invalid.");
         }
       }
 
-      function log(text) {
+      function log(text, style = "") {
         var date = new Date();
         var timestamp = [date.getHours(), date.getMinutes(), date.getSeconds()].join(":");
         var message = "[" + timestamp + "] " + text;
 
-        transcriptEl.appendFirst(createLogLineElement(message));
-        console.log(message);
+        transcriptEl.appendFirst(createLogLineElement(message, style));
+        console.log("%c"+message, style);
+      }
+
+      function displayError(text) {
+        log(text, 'color:red; font-family:helvetica,sans-serif; font-weight:bold; text-shadow:0 1px 2px rgba(0,0,0,.5);');
```

Comment:
Really don't care about IE that old

--
Colin Dean
http://cad.cx



On Mar 28, 2016, 10:12, at 10:12, Jason Khanlar <notifications@github.com> wrote:
>>          var date = new Date();
>>          var timestamp = [date.getHours(), date.getMinutes(),
>date.getSeconds()].join(":");
>>          var message = "[" + timestamp + "] " + text;
>>  
>> -        transcriptEl.appendFirst(createLogLineElement(message));
>> -        console.log(message);
>> +        transcriptEl.appendFirst(createLogLineElement(message,
>style));
>> +        console.log("%c"+message, style);
>> +      }
>> +
>> +      function displayError(text) {
>> +        log(text, 'color:red; font-family:helvetica,sans-serif;
>font-weight:bold; text-shadow:0 1px 2px rgba(0,0,0,.5);');
>
>Perhaps an UPPERCASE variable name may suffice. I see IE8/9/10 do not
>support JavaScript constants: http://caniuse.com/#search=const
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/pittmesh/ip-calculator/pull/18/files/90d14147ff28294a9746079af4012db62b1f650a..a82a1b8c2111c7c6f0bce73b8da2351dba548d23#r57575092